README.md

```diff
@@ -108,48 +108,48 @@ For feature requests, bugs, or technical questions, email us at [dev@buf.build][
 
 [badges_slack]: https://buf.build/links/slack
 [bash]: https://www.gnu.org/software/bash
-[binary]: https://buf.build/docs/installation#binary
-[breaking]: https://buf.build/docs/breaking/overview
-[breaking_rules]: https://buf.build/docs/breaking/rules
-[breaking_tutorial]: https://buf.build/docs/breaking/tutorial
+[binary]: https://buf.build/docs/cli/installation/#source
+[breaking]: https://buf.build/docs/breaking/overview/
+[breaking_rules]: https://buf.build/docs/breaking/rules/
+[breaking_tutorial]: https://buf.build/docs/breaking/tutorial/
 [brew]: https://brew.sh
-[bsr]: https://buf.build/docs/bsr/introduction
+[bsr]: https://buf.build/docs/bsr/
 [bsr_plugins]: https://buf.build/plugins
-[bsr_usage]: https://buf.build/docs/bsr/module/dependency-management
+[bsr_quickstart]: https://buf.build/docs/bsr/quickstart/
 [buf]: https://buf.build
 [build_config]: https://buf.build/docs/build/usage/#key-concepts
 [build_usage]: https://buf.build/docs/build/usage
-[cli-tutorial]: https://buf.build/docs/tutorials/getting-started-with-buf-cli
-[compiler]: https://buf.build/docs/reference/internal-compiler
-[docker]: https://buf.build/docs/installation#use-the-docker-image
+[cli-tutorial]: https://buf.build/docs/cli/quickstart/
+[compiler]: https://buf.build/docs/reference/internal-compiler/
+[docker]: https://buf.build/docs/cli/installation/#docker
 [docs]: https://buf.build/docs
 [email_dev]: mailto:dev@buf.build
 [email_info]: mailto:info@buf.build
 [features]: #features
 [fish]: https://fishshell.com
-[format_usage]: https://buf.build/docs/format/style
-[generate_usage]: https://buf.build/docs/generate/tutorial
-[ide]: https://buf.build/docs/editor-integration
+[format_usage]: https://buf.build/docs/format/style/
+[generate_usage]: https://buf.build/docs/generate/tutorial/
+[ide]: https://buf.build/docs/cli/editor-integration/
 [idl]: https://en.wikipedia.org/wiki/Interface_description_language
-[images]: https://buf.build/docs/reference/images
-[inputs]: https://buf.build/docs/reference/inputs
-[install]: https://buf.build/docs/installation
-[jetbrains]: https://buf.build/docs/editor-integration#jetbrains-ides
-[lint]: https://buf.build/docs/lint/overview
-[lint_rules]: https://buf.build/docs/lint/rules
-[lint_usage]: https://buf.build/docs/lint/tutorial
-[npm]: https://buf.build/docs/installation#npm
+[images]: https://buf.build/docs/reference/images/
+[inputs]: https://buf.build/docs/reference/inputs/
+[install]: https://buf.build/docs/cli/installation/
+[jetbrains]: https://buf.build/docs/cli/editor-integration/#jetbrains-ides
+[lint]: https://buf.build/docs/lint/overview/
+[lint_rules]: https://buf.build/docs/lint/rules/
+[lint_usage]: https://buf.build/docs/lint/tutorial/
+[npm]: https://buf.build/docs/cli/installation/#npm
```

Comment:
```suggestion
[npm]: https://buf.build/docs/cli/installation/#__tabbed_1_2
```
NPM should be this to correct the tab, or maybe the docs updated to use the ref `#npm`?

Reply:
Should be fixed in https://github.com/bufbuild/docs/pull/325 🎉